app/models/intake/ctc_intake.rb

```diff
@@ -345,14 +345,4 @@ def any_ip_pins?
   def filing_jointly?
     client.tax_returns.last.filing_status_married_filing_jointly?
   end
-
```

Comment:
i moved this from intake over to tax return because there was no good way to memoize the tax_return(2020) query without the contents getting stale, and loading the intake onto the tax return is simpler than loading one of many tax_returns on relation to intake.